app/views/propertyLinking/uploadEvidence.scala.html

```diff
@@ -59,108 +63,135 @@ <h2 class="govuk-error-summary__title" id="error-summary-heading">@messages("err
+                        @if(errors.nonEmpty){
+                            <span id="file-upload-1-error" class="govuk-error-message">
+                            @errors.map { error =>
+                                <span class="govuk-visually-hidden">Error:</span> @messages(error)
                             }
-                            </tbody>
-                        </table>
-                }
-                @formWithCSRF(action = UploadController.continue(EvidenceChoices.OTHER)) {
-                    @govukSelect(Select(
-                        id    = "evidenceType",
-                        name  = "evidenceType",
-                        items = SelectItem() +: EvidenceType.options.map(item =>
-                            SelectItem(
-                            value = Some(item),
-                            text  = messages(s"evidenceType.${item}"),
-                            selected = formEvidence.data.get("evidenceType").fold(false)(_ == item)
-                        )),
-                        label = Label(
-                            content = Text(messages("evidenceType.label"))
-                        ),
-                        errorMessage =  { formEvidence.error("evidenceType").map{ error =>
-                            ErrorMessage(content = Text(messages(error.message)))
-                        }}
-                    ))
-                    @govukButton(Button(
-                        content = Text(messages("uploadRatesBill.show.button")),
-                        attributes = Map("id" -> "continue")
+                            </span>
+                        }
+                        <div class="govuk-form-group">
+                            <label class="govuk-label--m" for="newFileButton">@messages("common.uploadFile.label")
+                            <span id="fileTypesText" class="govuk-hint">@messages("uploadFile.allowedTypes.l1")</span>
+                            </label>
+                            <input id="newFileButton" class="govuk-file-upload" name="file-upload-1" type="file">
+                        </div>
+                    </div>
+                }else{
+                    @govukWarningText(WarningText(
+                        iconFallbackText = "Warning",
+                        content = Text(messages("businessRatesAttachment.file.size.reached.max.limit"))
                     ))
                 }
 
+                <div class="message-warning govuk-visually-hidden" id="message-warning"><p class="govuk-body">@messages("fileupload.pleaseWait")</p></div>
+                <table class="govuk-table">
+                    <tbody class="govuk-table__body">
+                    @uploadedFiles.map { case (fileReference, upload) =>
+                    <tr class="govuk-table__row">
+                        <td class="govuk-table__cell">@{upload.fileMetadata.toDisplayFileName}</td>
+                        <td class="govuk-table__cell">@*(@upload.size.formatted)*@</td>
```

Comment:
File size is commented out ?

Reply:
🦅 👁️  I hadn't noticed that but I have checked now and  it was previously like that (not sure why). I will see if it can be removed all together.